app/src/main/java/com/junkfood/seal/ui/page/download/DownloadViewModel.kt

```diff
@@ -104,6 +104,9 @@ class DownloadViewModel @Inject constructor() : ViewModel() {
                 th.message ?: context.getString(R.string.unknown_error)
             )
             else if (isFetchingInfo) showErrorMessage(context.getString(R.string.fetch_info_error_msg))
+            else if (th.message!!.contains("Failed to find configured root")) showErrorMessage(
```

Comment:
Why don't we just ignore this `Throwable` and return?

Reply:
Okay! I wasn't sure if just returning the app would tell that the download was finished. Sorry for these things, it was too late and i was kinda slept 😆

Reply:
Ignoring the excpetion makes the UI not get updated to finish the download.

Reply:
Oh, finishProcessing()